src/Illuminate/Support/helpers.php

```diff
@@ -134,6 +134,26 @@ function env($key, $default = null)
     }
 }
 
+if (! function_exists('env_as_array')) {
+    /**
+     * Gets the value of an environment variable as array.
+     *
+     * @param  string  $key
+     * @param  string  $separator
+     * @param  array  $default
+     * @return array
+     */
+    function env_as_array(string $key, string $separator = ',', array $default = []): array
+    {
+        $value = env($key);
+        if (null === $value) {
+            return $default;
+        }
+
+        return explode($separator, $value);
```

Comment:
This may fail if value is null in future PHP versions. Explode should accept only strings